nisystemlink/clients/dataframe/models/_api_info.py

```diff
@@ -0,0 +1,19 @@
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class Operation(JsonModel):
```

Comment:
Should these classes have docstrings?

Reply:
Yes, thanks. flake8 isn't complaining because all of these modules are technically private 😕

Reply:
Nice. Do we have the ability to specify a docstring for the attributes too? I'm not sure what sphinx looks for.

Reply:
> I'm not sure what sphinx looks for.
I'm testing that right now actually. I just enabled the docs for the dataframe package to see how attributes work.

Reply:
I improved the generated docs for the DFS client and pulled in [autodoc-pydantic](https://autodoc-pydantic.readthedocs.io/en/stable/index.html) for the models. There's still more work we can do to clean up the docs, but I'll stop here for this PR.